src/main/scala/fi/oph/koski/valpas/repository/ValpasOppija.scala

```diff
@@ -123,5 +123,11 @@ case class ValpasHakutoive(
   hakukohdeNimi: Option[LocalizedString],
   hakutoivenumero: Option[Int],
   pisteet: Option[BigDecimal],
-  hyväksytty: Option[Boolean]
-)
+  minValintapisteet: Option[BigDecimal]
+) {
+  @SyntheticProperty
+  def hyväksytty: Option[Boolean] = (pisteet, minValintapisteet) match {
```

Comment:
päteehän tämä logiikka domainissa aina, vai voiko hyväksyminen perustua muuhunkin?

Reply:
Ratkaistaanko tämä sitten kun se tulee eteen?

Reply:
mietin, että tää voisi olla asia, joka pitäisi tehdä APIn toisella puolella, koska heillä on asiasta oikeampi domain-ymmärrys kuin Kosken puolella.

Reply:
muutokset siihen APIin tulee luultavasti olemaan vähän haastavia tätä ensivaihetta myöhemmin, koska tehdään eri tiimissä.

Reply:
Luultavasti juuri näin. Pitäis tehdä lappu taululle.

Reply:
Katsoin nyt API:n speksiä, siellä on kyllä kentät, joista tämä päättely pitäisi pystyä tekemään ilman tarvetta vertailla pisteitä Kosken puolella. Puuttuvatkohan nämä kentät vielä datoista?

![image](https://user-images.githubusercontent.com/6376268/112572855-c49b1800-8df3-11eb-91c4-826caaf1e502.png)